views/partials/product-education-video.hbs

```diff
@@ -0,0 +1,6 @@
+<aside class="product-education {{ theme }}">
+  {{#with (productEducation type)}}
+  <h2>{{ headingTxt }}</h2>
+  <video src="{{ videoSrc }}" width="3840" height="2160" type="video/mp4" controls></video>
```

Comment:
Should this have some sort of image poster?
I vaguely recall a `posterSrc` above, but not sure where/if it's being used.

```js
posterSrc: `${AppConstants.SERVER_URL}/img/education-vid-cover-${productType}.png`,
```

And not sure if those .png files are already in the repo, but I didn't see them above in the DIFFs when ctrl+F-ing.